scala-package/core/src/main/scala/org/apache/mxnet/Symbol.scala

```diff
@@ -101,7 +101,6 @@ class Symbol private(private[mxnet] val handle: SymbolHandle) extends WarnIfNotD
     var index: Int = -1
     for ((output, i) <- listOutputs().view.zipWithIndex) {
       if (output == name) {
-        require(index == -1, s"There are multiple outputs with name $name")
```

Comment:
I am not sure I understand why you need to remove this? I thought this one of the limitations that each node should have unique name

Reply:
I explained in the description of the PR:
> Regarding the change in Symbol.scala, it is a bug fix. Because we should allow users to expose two outputs but with same name. for example, https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/rnn/rnn_cell.py#L398